pyvista/_version.py

```diff
@@ -15,7 +15,7 @@
 
 """
 # major, minor, patch
-version_info = 0, 35, 'dev0'
+version_info = 0, 35, 0
```

Comment:
What we should do is bump to the next dev version in the source and then bump to `0.35.0` in the release branch.

```suggestion
version_info = 0, 36, 'dev0'
```

Reply:
Thanks for this. I reflected on this in #2911 .